outlook/src/utils/Misc.ts

```diff
@@ -0,0 +1,3 @@
+export function fieldEmpty(fieldName):boolean {
+    return (fieldName == null || fieldName == false || fieldName == 'false' || fieldName == '');
+}
```

Comment:
I don't see this function used anywhere?

Reply:
Yes, forgot to delete this one.